modules/administration/pages/openscap.adoc

```diff
@@ -8,12 +8,16 @@ It allows you to schedule and view compliance scans for any client.
 
 == About SCAP
 
-The Security Certification and Authorization Package (SCAP) is a standardized compliance checking solution for enterprise-level Linux infrastructures.
+The Security Content Automation Protocol (SCAP) is a synthesis of interoperable specifications derived from community ideas.
 It is a line of specifications maintained by the National Institute of Standards and Technology (NIST) for maintaining system security for enterprise systems.
 
 SCAP was created to provide a standardized approach to maintaining system security, and the standards that are used continually change to meet the needs of the community and enterprise businesses.
 New specifications are governed by NIST's SCAP Release cycle to provide a consistent and repeatable revision work flow.
-For more information, see http://scap.nist.gov/timeline.html.
+For more information, see:
+http://scap.nist.gov/timeline.html.
+https://csrc.nist.gov/projects/security-content-automation-protocol 
+https://www.open-scap.org/features/standards/
+https://ncp.nist.gov/repository?scap
```

Comment:
Better use a formal list to avoid strange formatting results:

```suggestion

* http://scap.nist.gov/timeline.html
* https://csrc.nist.gov/projects/security-content-automation-protocol 
* https://www.open-scap.org/features/standards/
* https://ncp.nist.gov/repository?scap
```
Do we actually need all the items?

Reply:
Customer mentioned those last 3 links as a base to have the definition and explanation modified. Not sure whether they really need to be added, it's also true that it might not add anything useful for the average user reading the documentation.
All in all, the list does make sense, so I'll accept this suggestion and if needed it can be removed afterwards.